YeomanTemplates/generator-choAdmin-boilerplate/generators/app/templates/README.md

```diff
@@ -0,0 +1,23 @@
+Generated typescript api client by NSwag's openApiToTypeScriptClient (using fetch)
```

Comment:
We should use a README similar to the one under `charity_registration`. I will talk to @hdpinto about what we want to have here.